toolchain/check/testdata/interop/cpp/no_prelude/bad_import.carbon

```diff
@@ -0,0 +1,96 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+// TIP: To test this file alone, run:
+// TIP:   bazel test //toolchain/testing:file_test --test_arg=--file_tests=toolchain/check/testdata/interop/cpp/no_prelude/bad_import.carbon
+// TIP: To dump output, run:
+// TIP:   bazel run //toolchain/testing:file_test -- --dump_output --file_tests=toolchain/check/testdata/interop/cpp/no_prelude/bad_import.carbon
+
+// --- fail_import_cpp.carbon
+
+library "[[@TEST_NAME]]";
+
+// CHECK:STDERR: fail_import_cpp.carbon:[[@LINE+4]]:1: error: imported API 'Cpp' not found [ImportNotFound]
+// CHECK:STDERR: import Cpp;
+// CHECK:STDERR: ^~~~~~
+// CHECK:STDERR:
+import Cpp;
```

Comment:
Can you please make this specifically point to `Cpp` requiring a `library`, or add a TODO to that effect?

Reply:
Added a dedicated diagnostic.